src/ui/profile_preferences.ui

```diff
@@ -84,7 +84,7 @@
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
-      <number>0</number>
+      <number>7</number>
```

Comment:
Please save the profile preferences with the first tab showing to avoid this unnecessary change.